app/packages/operators/src/built-in-operators.ts

```diff
@@ -740,58 +738,45 @@ class ClearSelectedLabels extends Operator {
     });
   }
   async execute({ state }: ExecutionContext) {
-    state.set(fos.selectedLabels, {});
+    state.set(fos.selectedLabels, []);
   }
 }
 
 export function registerBuiltInOperators() {
   try {
-    registerOperator(CopyViewAsJSON);
-    registerOperator(ViewFromJSON);
-    registerOperator(ReloadSamples);
-    registerOperator(ReloadDataset);
-    registerOperator(ClearSelectedSamples);
-    registerOperator(OpenAllPanels);
-    registerOperator(CloseAllPanels);
-    registerOperator(OpenDataset);
-    registerOperator(ClearView);
-    registerOperator(ClearSidebarFilters);
-    registerOperator(ClearAllStages);
-    registerOperator(RefreshColors);
-    registerOperator(ShowSelectedSamples);
-    registerOperator(ConvertExtendedSelectionToSelectedSamples);
-    registerOperator(SetSelectedSamples);
-    registerOperator(OpenPanel);
-    registerOperator(OpenAllPanels);
-    registerOperator(ClosePanel);
-    registerOperator(CloseAllPanels);
-    registerOperator(SetView);
-    registerOperator(ShowSamples);
-    // registerOperator(ClearShowSamples);
-    registerOperator(ConsoleLog);
-    registerOperator(ShowOutput);
-    registerOperator(SetProgress);
-    registerOperator(TestOperator);
-    registerOperator(SplitPanel);
-    registerOperator(SetSelectedLabels);
-    registerOperator(ClearSelectedLabels);
+    _registerBuiltInOperator(CopyViewAsJSON);
```

Comment:
Looking at the diff it looks like some built in operators are removed..?

Reply:
I have refactored the operator loading and move some of the loading code out of this file. I have updated `registerOperator` with `_registerBuiltInOperator` to bind to built-in plugin. However, I will check to make sure they are not gone from the list 👍